src/clients/rewriter.ts

```diff
@@ -48,76 +38,32 @@ export class Rewriter extends AppGraphQLClient {
     })
   }
 
-  public routesIndexFiles = (): Promise<RouteIndexFiles> =>
+  public exportRedirects = (next?: string): Promise<ExportResponse> =>
     this.graphql
-      .query<string[], {}>(
+      .query<ExportResponse, { next: string }>(
         {
           query: `
-      query RoutesIndexFiles {
+      query ListRedirects($next: String) {
         redirect {
-          indexFiles {
-            lastChangeDate
-            routeIndexFiles {
-              fileName
-              fileSize
+          listRedirects(next: $next) {
+            next
+            routes {
+              from
+              to
+              type
+              endDate
             }
           }
         }
       }
       `,
-          variables: {},
-        },
-        {
-          metric: 'rewriter-get-redirects-index-files',
-        }
-      )
-      .then(path(['data', 'redirect', 'indexFiles'])) as Promise<RouteIndexFiles>
-
-  public routesIndex = (fileName: string): Promise<RouteIndexEntry[]> =>
-    this.graphql
-      .query<string[], { fileName: string }>(
-        {
-          query: `
-      query RoutesIndex($fileName: String!) {
-        redirect {
-          index(fileName: $fileName) {
-            id
-            lastChangeDate
-          }
-        }
-      }
-      `,
-          variables: { fileName },
-        },
-        {
-          metric: 'rewriter-get-redirects-index',
-        }
-      )
-      .then(path(['data', 'redirect', 'index'])) as Promise<RouteIndexEntry[]>
-
-  public exportRedirects = (from: number, to: number): Promise<Redirect[]> =>
-    this.graphql
-      .query<Redirect[], { from: number; to: number }>(
-        {
-          query: `
-      query ListRedirects($from: Int!, $to: Int!) {
-        redirect {
-          list(from: $from, to: $to) {
-            from
-            to
-            type
-            endDate
-          }
-        }
-      }
-      `,
-          variables: { from, to },
+          variables: { next },
         },
         {
           metric: 'rewriter-get-redirects',
         }
       )
-      .then(path(['data', 'redirect', 'list'])) as Promise<Redirect[]>
+      .then(path(['data', 'redirect', 'listRedirects'])) as Promise<ExportResponse>
```

Comment:
We're trying to reduce ramda usage when there are native solutions, how about:
```suggestion
      .then(() => data?.redirect?.listRedirects) as Promise<ExportResponse>
```